desk/app/activity.hoon

```diff
@@ -140,22 +141,39 @@
   =?  cor  ?=(%6 -.old)
     (emit %pass /adjust-old-default %agent [our.bowl dap.bowl] %poke noun+!>(%adjust-old-default))
   =?  old  ?=(%6 -.old)  [%7 +.old]
-  ?>  ?=(%7 -.old)
+  =?  old  ?=(%7 -.old)  [%8 +.old]
+  ?>  ?=(%8 -.old)
   =.  state  old
+  :: ::  insert missing volume defaults to %base
+  :: ::
+  :: =/  base-volume
+  ::   (~(gut by volume-settings.state) [%base ~] *volume-map:a)
+  :: =.  volume-settings.state
+  ::   %+  ~(put by volume-settings.state)  [%base ~]
+  ::   (~(uni by base-volume) *volume-map:a)
```

Comment:
did we mean to have this uncommented?

Reply:
Oh yes, I deployed it on the test ship but forgot to push the commit. Thanks!